package.json

```diff
@@ -24,6 +24,9 @@
     "explorermobile >= 11",
     "android >= 4.4"
   ],
+  "nodemonConfig": {
+    "ignore": ["assets", "src"]
```

Comment:
i think you also need to put `"client"` in there as well

Reply:
you're right, I've renamed client to src. Will revert src to client.